<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>773</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Настольная игра &quot;Точки&quot;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>3000</width>
          <height>3000</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget_field">
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="textElideMode">
            <enum>Qt::ElideMiddle</enum>
           </property>
           <property name="cornerButtonEnabled">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_settings">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Настройки</string>
        </property>
        <widget class="QLineEdit" name="lineEdit_player1">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>50</y>
           <width>113</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Player_1</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_player2">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>100</y>
           <width>113</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Player_2</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_player1">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>50</y>
           <width>81</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Игрок 1:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_player2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>100</y>
           <width>81</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Игрок 2:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_field_size">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>170</y>
           <width>131</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Размеры поля:</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_width">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>180</y>
           <width>70</width>
           <height>25</height>
          </rect>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_height">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>180</y>
           <width>70</width>
           <height>25</height>
          </rect>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>180</y>
           <width>31</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>  X</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_start">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>240</y>
           <width>150</width>
           <height>35</height>
          </rect>
         </property>
         <property name="text">
          <string>Начать игру!</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="textEdit_report">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>290</y>
           <width>311</width>
           <height>75</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPreferences"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
